packages/ui/product/Product.tsx

```diff
@@ -0,0 +1,46 @@
+import { CardWrapper, CardDetails, CardComponent } from './styles';
+import StarSVG from './StarSVG';
+import { Title, Card, Image, Text, Button, Rating } from '@mantine/core';
+interface ProductProps {
```

Comment:
Interface name should start with `I` 

ex: `IProductProps`

Reply:
Added prefix